examples/2_intermediate/mta.ipynb

```diff
@@ -0,0 +1,332 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dacee7cb-d803-4eee-b135-c9ffd6ad2888",
+   "metadata": {},
+   "source": [
+    "In order to deal with the GTFS-rt data, we'll use the [nyct-gtfs](https://pypi.org/project/nyct-gtfs/) library, available from PyPI through\n",
+    "\n",
+    "```\n",
+    "pip install nyct-gtfs\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dcc5655d-3c45-4f96-842e-bb5f1b7d2127",
+   "metadata": {},
+   "source": [
+    "The MTA feed can be inspected as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9409305a-a84a-493f-8eeb-9a7c1c2c81eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nyct_gtfs import NYCTFeed\n",
+    "\n",
+    "# Load the realtime feed from the MTA site\n",
+    "feed = NYCTFeed(\"https://api-endpoint.mta.info/Dataservice/mtagtfsfeeds/nyct%2Fgtfs\", api_key=\"\")\n",
+    "\n",
+    "# feed_id must be a valid feed URL or one of: \n",
+    "# '1', '2', '3', '4', '5', '6', '7',\n",
+    "# 'S', 'GS', 'A', 'C', 'E', 'H',\n",
+    "# 'FS', 'SF', 'SR',\n",
+    "# 'B', 'D', 'F', 'M', 'G', 'J', 'Z', 'N', 'Q', 'R', 'W', 'L',\n",
+    "# 'SI', 'SS', 'SIR'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18e067c5-3f57-43a5-a744-a3bfa213c69b",
+   "metadata": {},
+   "source": [
+    "Let's explore the data first. `feed` is a `nyct_gtfs.feed.NYCTFeed` instance, with the most important methods being the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "349a3179-f49d-4be8-8f65-9599579695b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feed.refresh?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4d2d4de5-2a72-4c68-9618-106bda47c005",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feed.filter_trips?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9d89289d-ee10-4bee-a37d-0863fde7d1f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feed.trips?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bf0fe58-ced8-49a7-84bb-09b9abdb41ee",
+   "metadata": {},
+   "source": [
+    "In our case, we will (for simplicity) filter the trips to collect information only about the 1, 2 and 3 trains, and we will start with trains going through 34 St-Penn Station."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "429163be-6a02-4a14-b1f5-5f5016213af0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This cell can be run multiple times, and data will be refreshed every 30s\n",
+    "feed.refresh()\n",
+    "trains = feed.filter_trips(underway=True, headed_for_stop_id=['128', '128N'])\n",
```

Comment:
I wouldnt hardcode the stops in the adapter, rather stop_ids should be passed as an argument to the adapter so its more flexible.  The graph should then pass the stops its interested in

Reply:
Good suggestion! I'll fix it.

Reply:
this doesnt seem to be addressed in the latest commit